### PR TITLE
Update 1.0.1

### PR DESCRIPTION
**Pull request by Jrice1317**
Jira ticket: https://anaconda.atlassian.net/browse/PKG-590
Upstream repo: https://github.com/maxcountryman/flask-bcrypt

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,30 +15,37 @@ source:
...
   imports:
     - flask_bcrypt
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/maxcountryman/flask-bcrypt
+  home: https://flask-bcrypt.readthedocs.io/en/1.0.1/
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The correct home URL is ``https://github.com/maxcountryman/flask-bcrypt` https://github.com/maxcountryman/flask-bcrypt/blob/d9064521998846a59a98f311e3bd135e1dfe082c/setup.py#L17

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,30 +15,37 @@ source:
...
+  license: BSD-3-Clause
   license_file: LICENSE
   license_family: BSD
-  license: BSD 3-Clause
   summary: 'Bcrypt hashing for Flask.'
+  description: 'Flask-Bcrypt is a Flask extension that provides bcrypt hashing utilities for your application.'
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The single quote symbol `'` is not needed for the string. It's redundant

**Review by varlackc — APPROVED** [review]
not sure if we still need the `abs.yaml` file
https://github.com/AnacondaRecipes/flask-bcrypt-feedstock/blob/7c45cf816adc83eec023981b87322dcd0dc8f402/abs.yaml

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Ahhh okay, thanks for pointing that out! I'll be sure to check there for the home urls.

File: recipe/meta.yaml (hunk 2 above)
Comment (Jrice1317) [reply recipe/meta.yaml]: Okay, thanks! I was following the format of the summary, but I will remove the single quotes from both the summary and description.

**Comment by skupr-anaconda** [comment]
An error on linux-64:
`ImportError: libffi.so.7: cannot open shared object file: No such file or directory`
 
`bcrypt` requires `cffi` https://github.com/AnacondaRecipes/bcrypt-feedstock/blob/9fe4c181e439f7ef0e36057a1dac09129679f3a9/recipe/meta.yaml#L22 but the shared object cannot be found

**Comment by Jrice1317** [comment]
> An error on linux-64: `ImportError: libffi.so.7: cannot open shared object file: No such file or directory`
> 
> `bcrypt` requires `cffi` https://github.com/AnacondaRecipes/bcrypt-feedstock/blob/9fe4c181e439f7ef0e36057a1dac09129679f3a9/recipe/meta.yaml#L22 but the shared object cannot be found

I got this error locally after using the docker to emulate linux-64.
Screen Shot 2022-11-16 at 5 41 37 PM

It looks like cffi is being improperly installed, because it exists, but the local reference seems to be incorrectly named and also pointing to misspelled version (i.e. libffi.7.so should be libffi.so.7 and should be pointing to libffi.so.8).
Screen Shot 2022-11-16 at 5 43 04 PM

**Review by ryanskeith — COMMENTED** [review]
Let's figure out why this is dying linux-64.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -15,30 +15,37 @@ source:
...
 
 build:
-  noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<37]
 
 requirements:
   build:
```
Comment (cbouss) [inline recipe/meta.yaml]: This should be `host` and not `build`. That should remove some linter errors.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**